pkg/kubelet/config/config.go

```diff
@@ -519,6 +519,7 @@ func copyPods(sourcePods []*v1.Pod) []*v1.Pod {
 		pod, err := api.Scheme.Copy(source)
 		if err != nil {
 			glog.Errorf("unable to copy pod: %v", err)
+			continue
```

Comment:
Yes, I agree. Please apply the same fix to other places you see this pattern in this file. We should probably also try the type conversion and log and `continue` if that fails, rather than risk a panic.